src/user/views/UserDashboard.vue

```diff
@@ -0,0 +1,252 @@
+      <section class="events">
+        <header class="card--heading">
+          <h5>My Events</h5>
+        </header>
+
+        <!-- All Events -->
+        <div class="events--all">
+          <!-- Scheduled Events -->
+          <section class="event-status">
+            <header class="event-status--header">
+              <div class="subtitle1">Scheduled</div>
+            </header>
+            <ul class="dashboard-events">
+              <li v-for="event in events" :key="event.id">
+                <template v-if="event.isScheduled">
+                  <DashboardEvent :event="event" />
+                </template>
+              </li>
+            </ul>
+          </section>
+
+          <!-- Unscheduled Events -->
+          <section class="event-status">
+            <header class="event-status--header">
+              <div class="subtitle1">Unscheduled</div>
+            </header>
+            <ul class="dashboard-events">
+              <li v-for="event in events" :key="event.id">
+                <template v-if="!event.isScheduled">
+                  <DashboardEvent :event="event" />
+                </template>
+              </li>
+            </ul>
+          </section>
+        </div>
+      </section>
+    </div>
+
+    <!-- Right Calendar Card -->
+    <div class="card--schedule card">
+      <section class="schedule">
+        <header class="card--heading">
+          <h5>My Set Schedule</h5>
+        </header>
+        <Calendar
+          v-model:calendar="calendar"
+          :start-time="startTime"
+          :end-time="endTime"
+          :edit-mode="isEditable"
+          class="schedule--calendar"
+        />
+        <div class="schedule--subsection">
+          <div class="schedule--timezone caption">
+            (Time displayed in PST—Vancouver)
+          </div>
+          <button
+            class="schedule--edit button"
+            type="button"
+            @click="toggleEdit"
+          >
+            {{ isEditable ? "View Schedule" : "Edit Schedule" }}
+          </button>
+        </div>
+      </section>
+    </div>
+  </main>
+</template>
+
+<script lang="ts">
+// TODO: Fix Calendar select
+// TODO: Prevent potential overflow of calendar in zoom/mobile
+// TODO: Change card width (flex-grow) when wrapped for mobile/smaller screens
+// TODO: Figure out "Cannot find name 'Calendar'.Vetur(2304)"
+
+import { defineComponent, computed, reactive, ref } from "vue";
+import DashboardEvent from "@/user/components/DashboardEvent.vue";
+import Calendar from "@/calendar/components/Calendar.vue";
+
+const start = new Date("April 1, 2021 09:00:00");
+const end = new Date("April  7, 2021 21:30:00");
+
+export default defineComponent({
+  name: "UserDashboard",
+
+  components: { DashboardEvent, Calendar },
+
+  props: {},
+
+  setup() {
+    const startTime = computed(() => start.toISOString());
+    const endTime = computed(() => end.toISOString());
+
+    const calendar = reactive({
+      blocks: []
+    });
```

Comment:
> currently can select individual block and have it display after another render like clicking the "Edit/View Schedule" button

This tells me that the calendar works, it's just not reactive to the change i.e. the data is updated, but the view doesn't rerender.

I think the reason the `Calendar` isn't working is because you're using `reactive` instead of `ref`. When you use `reactive`, you're creating an object whose _properties_ are reactive, but the object itself isn't.

Simply changing it to `ref` should make it work fine:

```suggestion
    const calendar = ref({
      blocks: []
    });
```